tests/lib/log/log.go

```diff
@@ -0,0 +1,59 @@
+package log
```

Comment:
It is a file logger :-). The intention was not to build an awesome log abstraction, perhaps the name is wrong. I used the method Log() because I kept in mind the t.Log() method, was not thinking about the Go log package actually.

About the CI, it seems that it will have to be an option to use stdout, I'm going to work on it and take a look on the go log package to build something better.